Household.cat

```diff
@@ -0,0 +1,5657 @@
+                    <constraint field="selections" scope="parent" value="1.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="ccd7-98dd-3872-5508" type="min"/>
+                  </constraints>
+                  <categoryLinks>
+                    <categoryLink id="8bd3-3f0e-31ca-11cc" name="Agile" hidden="false" targetId="e580-b213-3f86-c2fc" primary="false"/>
+                    <categoryLink id="714b-d216-03fd-47aa" name="Knight" hidden="false" targetId="184a-ded9-ae1a-e357" primary="false"/>
+                  </categoryLinks>
+                  <selectionEntryGroups>
+                    <selectionEntryGroup id="3bcd-64ae-2b1b-f0cb" name="Arms" hidden="false" collective="false" import="true">
+                      <constraints>
+                        <constraint field="selections" scope="parent" value="2.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="89cd-74b7-8218-c794" type="min"/>
+                        <constraint field="selections" scope="parent" value="2.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="1a89-4594-ff81-3c39" type="max"/>
+                      </constraints>
+                      <entryLinks>
+                        <entryLink id="ad96-42d4-3c3a-5504" name="Lightning Cannon" hidden="false" collective="false" import="true" targetId="c240-3acf-9c72-43d1" type="selectionEntry">
+                          <constraints>
+                            <constraint field="selections" scope="parent" value="2.0" percentValue="false" shared="false" includeChildSelections="false" includeChildForces="false" id="d960-eeaa-9ce9-2927" type="min"/>
+                          </constraints>
+                        </entryLink>
+                      </entryLinks>
+                    </selectionEntryGroup>
+                    <selectionEntryGroup id="2e99-cf1d-e924-675b" name="Upgrades" hidden="false" collective="false" import="true">
+                      <entryLinks>
+                        <entryLink id="e274-8998-2666-38ed" name="Phased Plasma-Fusil" hidden="false" collective="false" import="true" targetId="006e-1fd0-7707-a241" type="selectionEntry"/>
+                      </entryLinks>
+                    </selectionEntryGroup>
+                  </selectionEntryGroups>
+                  <costs>
+                    <cost name=" Stratagem Points" typeId="efbf-52f7-fd08-f329" value="0.0"/>
+                    <cost name=" Points" typeId="a731-e220-2d8a-41bf" value="45.0"/>
+                  </costs>
+                </selectionEntry>
+                <selectionEntry id="b09f-9393-7c1b-10f9" name="Questoris Knight Lord Magaera" hidden="false" collective="false" import="true" type="model">
+                  <constraints>
+                    <constraint field="sel
```

Comment:
missing a word at the end?

Reply:
Same line as above? Maybe your marker ended up off?